src/components/CirclePagination/CirclePagination.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FC } from "react";
+import styled from "styled-components";
+import { Row, Box } from "../../theme/custom-styled-components";
+import { ColorType } from "../../theme/util";
+import Circle from "./Circle";
+
+export interface CirclePaginationProps {
+  activeItem: number;
+  data: any;
+  bgColor?: ColorType;
+  onItemClick: (index: any) => void;
+  className?: string;
+}
+
+const ButtonStyled = styled(Box)`
+  background: none;
+  border: none;
+  cursor: pointer;
+  padding: 0;
+`;
+
+export const CirclePagination: FC<CirclePaginationProps> = ({
+  onItemClick,
+  data,
+  bgColor,
+  activeItem,
+  className,
+}: CirclePaginationProps) => {
+  return (
+    <Row className={className} mt={{ xxs: 3, lg: 5 }}>
+      {data.map((item: any, index: number) => {
+        return (
+          <ButtonStyled
+            aria-label={`carousel-page-${index}`}
+            as="button"
+            key={`pagination-element-${index}-${item.toString()}`}
```

Comment:
The Template literal is already a string literal, so we don't need `toString()` method. Please remove it.

Reply:
my intention was to create a unique string for the `item`, which is one element of the data object. I did it to create a unique key, because the contents of the data item are unknown